ingestion-beam/src/main/java/com/mozilla/telemetry/options/OutputType.java

```diff
@@ -161,6 +162,8 @@
               BigQueryIO.writeTableRows().withMethod(BigQueryIO.Write.Method.STREAMING_INSERTS)
                   .withCreateDisposition(BigQueryIO.Write.CreateDisposition.CREATE_NEVER)
                   .withWriteDisposition(BigQueryIO.Write.WriteDisposition.WRITE_APPEND)
+                  .skipInvalidRows()
+                  .withFailedInsertRetryPolicy(InsertRetryPolicy.retryTransientErrors())
```

Comment:
Without setting this retry policy, any failed insert bubbled up to a pipeline execution exception rather than getting captured in getFailedInserts. I asked about this on the beam user list since it doesn't seem to be documented behavior.